lib/logstasher/action_view/log_subscriber.rb

```diff
@@ -51,7 +51,8 @@ def store
       def event_data(event)
         {
           name: event.name,
-          transaction_id: event.transaction_id
+          transaction_id: event.transaction_id,
+          allocations: event.try(:allocations) || 0
```

Comment:
Maybe we could add the `allocations` key/value only if `allocations` is available in event to avoid having 0 allocations? Or use `nil` to have less wrong informations when no `#allocations` accessibles?

Reply:
Do you mean something like that: https://github.com/honestica/logstasher/compare/action_view_allocations...honestica:action_view_allocations_conditional?expand=1

I'm thinking the tests should then include the Active Record version context.

Reply:
Yep